commons/src/main/java/org/openmrs/eip/camel/OauthProcessor.java

```diff
@@ -64,15 +64,15 @@ public void process(Exchange exchange) throws Exception {
 				}
 				
 				long currentSeconds = Utils.getCurrentSeconds();
-				Map response = producerTemplate.requestBody(OAUTH_URI, null, Map.class);
+				var response = producerTemplate.requestBody(OAUTH_URI, null, Map.class);
```

Comment:
Funny, I'm not familiar with this Java 10+ feature, but if we know better already, isn't it better to already narrow down the type?

Reply:
`var` is fully typed, inferred based on the right-hand side of the equation, but `response` ends up as `Map<?, ?>` or w/e. So this isn't creating a variable that can hold anything, it just allows you to type:

```
var myMap = new HashMap<String, String>();
```

Instead of

```
Map<String, String> myMap = new HashMap<>();
```